hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -41,6 +41,261 @@
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_sigverify_always_a_nothing_b_bad_dev
+      // We can use the same vseq as the `*_a_bad_b_bad_*` tests above, as we expect the same ROM boot fault value.
+      uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_b_corrupted:1:ot_flash_binary:signed:fake_dev_key_0:no_sw_log_db",
+        "//sw/device/silicon_creator/rom/e2e:otp_img_sigverify_always_dev:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+sw_test_timeout_ns=50_000_000",
+        "+use_otp_image=OtpTypeCustom",
+        "+bypass_alert_ready_to_end_check=1",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_sigverify_always_a_nothing_b_bad_prod
+      // We can use the same vseq as the `*_a_bad_b_bad_*` tests above, as we expect the same ROM boot fault value.
+      uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_b_corrupted:1:ot_flash_binary:signed:fake_prod_key_0:no_sw_log_db",
+        "//sw/device/silicon_creator/rom/e2e:otp_img_sigverify_always_prod:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+sw_test_timeout_ns=50_000_000",
+        "+use_otp_image=OtpTypeCustom",
+        "+bypass_alert_ready_to_end_check=1",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_sigverify_always_a_nothing_b_bad_prod_end
+      // We can use the same vseq as the `*_a_bad_b_bad_*` tests above, as we expect the same ROM boot fault value.
+      uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_b_corrupted:1:ot_flash_binary:signed:fake_prod_key_0:no_sw_log_db",
+        "//sw/device/silicon_creator/rom/e2e:otp_img_sigverify_always_prod_end:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+sw_test_timeout_ns=50_000_000",
+        "+use_otp_image=OtpTypeCustom",
+        "+bypass_alert_ready_to_end_check=1",
+      ]
+      reseed: 1
+      run_timeout_mins: 120
+    }
+    {
+      name: rom_e2e_sigverify_always_a_nothing_b_bad_rma
+      // We can use the same vseq as the `*_a_bad_b_bad_*` tests above, as we expect the same ROM boot fault value.
+      uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_b_corrupted:1:ot_flash_binary:signed:fake_prod_key_0:no_sw_log_db",
+        "//sw/device/silicon_creator/rom/e2e:otp_img_sigverify_always_rma:4",
+      ]
+      en_run_modes: ["sw_test_mode_rom"]
+      run_opts: [
+        "+sw_test_timeout_ns=50_000_000",
+        "+use_otp_image=OtpTypeCustom",
+        "+bypass_alert_ready_to_end_check=1",
```

Comment:
It will be great if we could exit the test cleanly, i.e. with all alerts quashed. The reason for that is if we leave DUT in a state with some on going activity, we end up with spurious end-of-sim assertion errors when things (like constant fatal alerts firing for example) have to cross clock domains and the sim abruptly ends before the assertion that track these events complete. A simple fix for pending alerts at the end of sim is to just call `apply_reset` (which will wiggle the POR_N) signal and reset everything) - it will make the test more robust in terms of nightly triage overheads.

Reply:
Ah sg, yeah these ROM E2E test that boot fault generate a fatal alert (via the SW request mechanism in rv_core_ibex). I will add that in the cleanup PR too. Thanks!

Reply:
Addressed in #16601.